exercises/scripts/createE2ETestWorkflow.ts

```diff
@@ -0,0 +1,79 @@
+import { argv } from 'process'
+import { writeFile } from 'fs'
+
+const EXERCISES = ['00-init']
+const WORKFLOW_FILE = '../../.github/workflows/main.yml'
+
+const HEADER = `name: End-to-end tests
```

Comment:
just to address my OCD and to make it more visually clear - could you add this "name: end-to-end test" on a new line? If you're afraid of an empty initial line, you can use backslash and the end of the line

Reply:
Fixed and renamed, see ff95851 and efff410